apps/root/src/pages/dca/positions/components/usd-dashboard/index.tsx

```diff
@@ -425,212 +270,107 @@ const UsdDashboard = ({ selectedChain, onSelectTokens, selectedTokens }: UsdDash
+                      value={positionCountLabel.relativeLengthToShow}
+                      valueBuffer={positionCountLabel.relativeLength}
+                      fill={positionCountLabel.fill}
                     />
-                  </Pie>
-                </PieChart>
-              </ResponsiveContainer>
-            </Grid>
-          </Hidden>
-          <StyledGrid item xs={12} md={7}>
-            <StyledLabelContainer>
-              {tokensCountLabels.map((positionCountLabel) => (
-                <Grid
-                  container
-                  alignItems="center"
-                  onMouseOut={() => {
-                    onSelectTokens(null);
-                    setShowPopper(false);
-                  }}
-                  onMouseOver={(event) => {
-                    onSelectTokens(positionCountLabel.tokens ? positionCountLabel.tokens : [positionCountLabel.name]);
-                    if (positionCountLabel.isOther) {
-                      handlePopperEl(event);
-                    }
-                  }}
-                  sx={{ cursor: 'pointer' }}
-                  key={positionCountLabel.name}
-                >
-                  <Grid item xs={1}>
-                    <StyledBullet fill={positionCountLabel.fill} />
-                  </Grid>
-                  <Grid item xs={3}>
-                    <StyledTypography variant="bodySmall" disabled={positionCountLabel.summedBalanceToShow <= 0n}>
-                      {positionCountLabel.name}
-                    </StyledTypography>
-                  </Grid>
-                  <Grid item xs={5}>
-                    {positionCountLabel.isOther && (
-                      <Popper id="other-popper" open={showPopper} anchorEl={anchorEl}>
-                        <DashboardPopper tokensBreakdown={positionCountLabel.tokensBreakdown} />
-                      </Popper>
-                    )}
-                    {!positionCountLabel.isOther && (
-                      <BorderLinearProgress
-                        variant="buffer"
-                        value={positionCountLabel.relativeLengthToShow}
-                        valueBuffer={positionCountLabel.relativeLength}
-                        fill={positionCountLabel.fill}
-                      />
-                    )}
-                  </Grid>
-                  <Grid item xs={3} sx={{ textAlign: 'right' }}>
-                    <Typography variant="bodySmall">
-                      {!positionCountLabel.isOther
-                        ? formatCurrencyAmount(positionCountLabel.summedBalanceToShow, positionCountLabel.token, 4)
-                        : `$${positionCountLabel.summedBalanceUsdToShow.toFixed(2)}`}
-                    </Typography>
-                  </Grid>
+                  )}
+                </Grid>
+                <Grid item xs={3} sx={{ textAlign: 'right' }}>
+                  <Typography variant="bodySmall">{`$${positionCountLabel.value.toFixed(2)}`}</Typography>
```

Comment:
template literal is not necessary. Also, could textAlign be applied directly to the Typography component?

Reply:
I'll remove the template literal but the text align has to stay since typography is inline-block and cannot use the textAlign